service/src/main/kotlin/fi/espoo/evaka/placement/PlacementControllerCitizen.kt

```diff
@@ -150,6 +156,18 @@ class PlacementControllerCitizen(
                 )
             }
         }
-        Audit.PlacementTerminate.log(targetId = body.unitId, objectId = body.type)
+
+        val placements =
+            terminatablePlacementGroup.placements + terminatablePlacementGroup.additionalPlacements
+
+        Audit.PlacementTerminate.log(
+            targetId = listOf(body.unitId, body.type, childId),
+            objectId = placements.map { it.id } + cancelableTransferApplicationIds,
+            meta =
+                mapOf(
+                    "placementIds" to placements.map { it.id },
+                    "transferApplicationIds" to cancelableTransferApplicationIds
+                )
+        )
```

Comment:
Something like this would be more idiomatic Kotlin:

```kotlin
db.connect { dbc ->
    val terminatablePlacementGroup = dbc.read { ... }
    val cancelableTransferApplicationIds = dbc.transaction { ... }
    Pair(terminatablePlacementGroup, cancelableTransferApplicationIds)
}.also { (terminatablePlacementGroup, cancelableTransferApplicationIds) ->
    val placements = terminatablePlacementGroup.placements + terminatablePlacementGroup.additionalPlacements
    Audit.PlacementTerminate.log(...)
}
```